src/GithubMergeTool/GithubMergeTool.cs

```diff
@@ -180,9 +182,40 @@ git commit
             jsonBody = JObject.Parse(await response.Content.ReadAsStringAsync());
 
             var prNumber = (string)jsonBody["number"];
+            var mergeable = (bool?)jsonBody["mergeable"];
+            if (mergeable == null)
```

Comment:
:( is the mergeable attribute computed asynchronously?

Reply:
correct, it takes two seconds or so for dotnet/roslyn, but probably depends on complexity of changes